### PR TITLE
Clean up controller-manager.

### PR DESCRIPTION
**Pull request by bjhaid**
These are based on recommendation from
[staticcheck](http://staticcheck.io/).

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: cmd/controller-manager/app/options/generic.go [hunk 1]
```diff
@@ -103,9 +103,7 @@ func (o *GenericControllerManagerConfigurationOptions) Validate(allControllers [
 		if controller == "*" {
 			continue
 		}
-		if strings.HasPrefix(controller, "-") {
-			controller = controller[1:]
-		}
+		strings.TrimPrefix(controller, "-")
 		if !allControllersSet.Has(controller) {
```
Comment (stewart-yu) [inline cmd/controller-manager/app/options/generic.go]: does this fix necessary?
if necessary, IMO，we should change like this:
```
controller=strings.TrimPrefix(controller, "-")
```
but in this way, the cost is greater than change before.
@baasbank WDYT?

**Review by stewart-yu — COMMENTED** [review]
one nil

File: cmd/controller-manager/app/options/generic.go (hunk 1 above)
Comment (bjhaid) [reply cmd/controller-manager/app/options/generic.go]: ah thanks, will fix shortly

Comment (bjhaid) [reply cmd/controller-manager/app/options/generic.go]: fixed

**Pull request merged**